index.html

```diff
@@ -0,0 +1,166 @@
+                    <h2 class="contacts-title"> <i class="fas fa-address-book"></i>  Contacts </h2>
+                     <ul>
+                        <li class="contacts-item">
+                            <h3> Phone: </h3> <a href="tel:+48788249945" class="contacts-link"> +48 788 249 945 </a>
+                        </li>
+                        <li class="contacts-item">
+                            <h3> Email: </h3> <a href="mailto:rostyslavlogvinov@gmail.com" class="contacts-link"> rostyslavlogvinov@gmail.com</a>
+                        </li>          
+                    </ul>
+                    <div>
+                        <a href="https://github.com/RostyslavL"  class="social-media-link">
+                            <i class="fab fa-github"></i>
+                        </a>
+                        <a href="https://www.facebook.com/rostyslav.logvinov" class="social-media-link">
+                            <i class="fab fa-facebook"></i>
+                        </a>
+                        <a href="https://www.linkedin.com/in/rostyslav-l-815b271b0/" class="social-media-link">
+                            <i class="fab fa-linkedin-in"></i>
+                        </a>
+                    </div>
+                </section>
+            </div>
+            <div class="container-right"> 
+            <main>
+                    <!-- Education Section -->
+                        <section class="section">
+                            <h2 class="education-heading "> <i class="fas fa-user-graduate "></i>  Education </h2>
+                            <hr />
+                                <div>
+                                    <h2><i class="fas fa-university"></i>   Gdansk University of Technology, Gdansk </h2>
+                                    <details><summary> <strong> Master Degree </strong>   </summary>       
+                                        <ul class="study-skills-text">
+                                            <li> <h5> <b>Field of Study </b> :</h5> Electronics and Telecommunications       </li>
+                                            <li> <h5> <b>Scope of Study </b> :</h5> Teleinformation Networks and Systems     </li>
+                                            <li> <h5> <b>Discipline     </b> :</h5> Information and Communication Technology </li>
+                                            <li> <h5> <b>Terms of study </b> :</h5> 28.02.2017 - 19.11.2019                  </li>
+                                        </ul>   
+                                    </details> 
+                                    <h2><i class="fas fa-university"></i>  COLLEGE OF COMMUNICATIONS ,KYIV </h2> 
+                                    <details><summary><strong>  Engineer Degree </strong> </summary> 
+                                        <ul class="study-skills-text">
+                                            <li> <h5> <b> Field of study </b>: </h5> Telecommunications                       </li>
```

Comment:
looks like `<b>` is not needed here at all, font should be already in bold due to `h5`